### ISSUE TITLE
cascade_invocations false positives on top level functions in prefixed library

### ISSUE THREAD

**Issue by alexeieleusis** [dart-lang/linter#350]
```
library_name.function1();
library_name.function2();
```

**Comment by alexeieleusis** [comment]
it also happens with top level functions
